### PR TITLE
Patch biowulf config

### PR DESCRIPTION
**Pull request by qiyubio**
---
name: Update the cluster config
about: patches
---

Please follow these steps before submitting your PR:

- [x] Your PR targets the `master` branch
- [x] You've included links to relevant issues, if any

### CODE REVIEW

**Review by jfy133 — APPROVED** [review]

File: conf/biowulf.config [hunk 1]
```diff
@@ -6,48 +6,41 @@ params {
...
     autoMounts = true
-    cacheDir = "/data/$USER/singularity"
-    envWhitelist='https_proxy,http_proxy,ftp_proxy,DISPLAY,SLURM_JOBID,SINGULARITY_BINDPATH'
+    cacheDir = "/data/$USER/nxf_singularity_cache"
+    envWhitelist='https_proxy,http_proxy,ftp_proxy,DISPLAY,SLURM_JOB_ID,SINGULARITY_BINDPATH'
 }
 
 env {
-    SINGULARITY_CACHEDIR="/data/$USER/singularity"
+    SINGULARITY_CACHEDIR="/data/$USER/.singularity"
```
Comment (jfy133) [inline conf/biowulf.config:21]: Just noticed this, shouldn't this be the same as the path above?

File: conf/biowulf.config [hunk 2]
```diff
@@ -3,51 +3,44 @@ params {
...
```
Comment (edmundmiller) [inline conf/biowulf.config:18]:
```suggestion
    SINGULARITY_CACHEDIR="/data/$USER/nxf_singularity_cache"
```

**Review by edmundmiller — DISMISSED** [review]
I have only seen them ever be the same. 

So I think, make them match or reveal your secrets 😁

**Comment by qiyubio** [comment]
There is bug in this setting, the time without .h will cause user's slurm job submitted with unlimited time, and the job will never got started since it excessed the wall time of slurm partition. Could my pull request got approved? Which has the newest resource-limits, and SINGULARITY_CACHEDIR is removed. 

 `   resourceLimits = [
        memory: 224,
        cpus: 32,
        time: 72
    ]
`

**Pull request merged**